docs/apache-airflow/authoring-and-scheduling/timetable.rst

```diff
@@ -217,3 +228,24 @@ is immediately triggered after you re-enable the DAG.
 
 By these examples, you see how `CronTriggerTimetable`_ triggers DAG runs is more intuitive and more similar to what
 people expect cron to behave than how `CronDataIntervalTimetable`_ does.
+
+
+.. _Differences between the two data intervals timetables:
+
+Differences between the two data intervals timetables:
```

Comment:
This title is not right since all timetables have a data interval. I would say something like _Differences between using a cron and timedelta to express data intervals_.

Reply:
That's the only two that are name `...DataIntervalTimetables`. But no problem, I'll update it.